workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py

```diff
@@ -82,6 +89,7 @@ def plot_2d_matplotlib(
             logging.info(f"plotting {varname} in {run}")
             v = run_diags.get_variable(run, varname)
             long_name_and_units = f"{v.long_name} [{v.units}]"
+            variable_long_names[varname] = v.long_name
```

Comment:
Nit: set this once instead of repeatedly?

Reply:
Done. Added a `long_names` property to `RunDiagnostics`.